### ISSUE TITLE
Help: How to keep internal port mapping without exposing the port to the host

### ISSUE THREAD

**Issue by ElleshaHackett** [qdm12/private-internet-access-docker#162]
**TLDR**: I want to access a mapped port through a web proxy but without exposing the port to the host.

1. Is this urgent?

    - [ ] Yes
    - [x] No

2. What VPN service provider are you using?

    - [x] PIA
    - [ ] Mullvad
    - [ ] Windscribe

3. What's the version of the program?

    Latest Docker image.

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

I've removed most of the sensitive info below so this config won't work out-of-the-box. But this config generally works for me. 

I have now `host:80 > proxy:80 > pia:9091 > transmission:9091` **and** `host:9091 > pia:9091 > transmission:9091` but I want:

- disable `host:9091 > pia:9091 > transmission:9091`
- only keep `host:80 > proxy:80 > pia:9091 > transmission:9091`

I think this mapping by the pia container is only done because its port is also exposed to the host. My point is exactly that I want to disable the direct port exposure on the host but want to internally keep `pia:9091 > transmission:9091` so that the proxy can access it. 

How do I set that up? Any ideas?

Configuration file:

```yml
proxy:
  container_name: proxy
  image: jwilder/nginx-proxy
  ports:
    - '80:80'
  networks:
    - proxy
pia:
  cap_add:
    - NET_ADMIN
  container_name: pia
  environment:
    - VIRTUAL_HOST=transmission.server.home
    - VIRTUAL_PORT=9091
  ports:
    - '9091:9091/tcp'                       # <<<< this I want to remove
  image: qmcgaw/private-internet-access
  networks:
    - proxy
  depends_on:
    - proxy
transmission:
  image: linuxserver/transmission
  container_name: transmission
  network_mode: 'service:pia'
  depends_on:
    - pia
    - proxy
```

Host OS:
Latest Zorin OS, basically Ubuntu.

**Comment by qdm12** [comment]
If proxy and pia are in the same Docker network, you should be able to reach port 9091 on pia from your proxy container without publishing the port. Or is it not working?

For example, in my pia container I have ports published AND an expose block (really for reminding me of the ports, I don't think it's needed):

```yml
    ports:
      - 8388:8388/tcp
      - 8388:8388/udp
    expose:
      - 8112/tcp
```

Such that my reverse proxy can access pia:8112 directly and it has been working great :wink:

**Comment by qdm12** [comment]
Hello, is it working for you? Can I close the issue? Thanks!
